raiden/network/proxies/token_network.py

```diff
@@ -1161,7 +1142,6 @@ def update_transfer(
         )
 
         if gas_limit:
-            error_prefix = 'updateNonClosingBalanceProof call failed'
             transaction_hash = self.proxy.transact(
```

Comment:
I just realized this is not protected with the channel operations lock (e.g. https://github.com/raiden-network/raiden/blob/0e8fa93a275deb2a5d33eaa8a377273fc5049559/raiden/network/proxies/token_network.py#L727-L729)

I have to think a bit to see if we do or don't need it here.